Mathlib/Order/WithBot.lean

```diff
@@ -1067,6 +1069,14 @@ theorem ofDual_map (f : αᵒᵈ → βᵒᵈ) (a : WithBot αᵒᵈ) :
   rfl
 #align with_bot.of_dual_map WithBot.ofDual_map
 
+lemma forall_lt_iff_eq_bot [Preorder α] {x : WithBot α} :
+    (∀ y : α, x < y) ↔ x = ⊥ := by
+  refine ⟨fun h ↦ ?_, fun h ↦ h ▸ fun y ↦ WithBot.bot_lt_coe y⟩
+  by_contra h'
```

Comment:
This is now very short with `eq_bot_iff_forall`, right?
Something like (untested) `exact eq_top_iff_forall.mpr (fun x \mapsto (h x).ne')`

Reply:
Maybe it would also be nice to unformize whether `x = ⊥` is LHS or RHS, but I don't have a strong opinion there.

Reply:
Done. Thanks.